crates/curp/src/server/storage/wal/pipeline.rs

```diff
@@ -34,6 +34,9 @@ pub(super) struct FilePipeline {
     stopped: Arc<AtomicBool>,
     /// Join handle of the allocation task
     file_alloc_task_handle: Option<JoinHandle<()>>,
+    #[cfg_attr(not(madsim), allow(unused))]
```

Comment:
Please use `#[cfg(madsim)]` to replace it.

Reply:
done